model/components/opencensus.yml

```diff
@@ -0,0 +1,31 @@
+---
+components:
+
+  # opencensus instrumentation toolkit
+  - id: opencensus
+    name: OpenCensus
+    description: A single distribution of libraries that automatically collects traces and metrics from your app, displays them locally, and sends them to any analysis tool.
```

Comment:
"... and sends them to any analysis tool" I would rather write something like: "... and provides means to write to different analysis tools"

Reply:
ok changed it ;) Was the official text from their website ^^